src/SDKs/TrafficManager/Management.TrafficManager/generate.cmd

```diff
@@ -4,7 +4,7 @@
 ::
 
 @echo off
-set autoRestVersion=0.16.0
+set autoRestVersion=0.13.2
```

Comment:
@vladca can you use the latest or at least 1.0 version of AutoRest.
Also can you use the commit Id as part of the argument in generate.cmd